HoloPipelines/pipelines/pipelineAbdomenDicom2glb.py

```diff
@@ -0,0 +1,26 @@
+from components import compDicom2nifti
+from components import compHttpRequest
+from components import compNifti2numpy
+from components import compSeparateNumpy
+from components import compNumpyTransformation
+import sys
+
+
+def main(inputDicomPath, outputGlbFolderPath):
+    generatedNiftiPath = compDicom2nifti.main(inputDicomPath, "_temp.nii")
+    segmentedNiftiPath = compHttpRequest.sendFilePostRequest(
+        "http://localhost:5000/model",
+        generatedNiftiPath,
+        "_tempAbdomenSegmented.nii.gz",
```

Comment:
Think about how to handle all the residue files leftover from the processes

Reply:
nii file now gets deleted in compNifti2numpy if stated (`deleteNiftiWhenDone` argument in compNifti2numpy. The argument is set to False by default)